Bio/Blast/NCBIXML.py

```diff
@@ -355,7 +355,7 @@ def _end_Parameters_gap_extend(self):
                                          int(self._value))
 
     def _end_Parameters_filter(self):
-        """Filtering options (-F) (PRIVATE)."""
+        """Record a filtering options (-F) (PRIVATE)."""
```

Comment:
Options is plural, so "Record a filtering options" --> "Record filtering options"?

Reply:
Well, this was pretty dumb xD

Reply:
Not really. You've been editing so many of these that little mistake can easily happen.